src/node/rpc/node_frontend.h

```diff
@@ -1807,6 +1807,29 @@ namespace ccf
         .set_forwarding_required(endpoints::ForwardingRequired::Never)
         .set_auto_schema<void, nlohmann::json>()
         .install();
+
+      auto process = [this](auto& args, const nlohmann::json& params) {
+        auto node_configuration_subsystem =
+          this->context.get_subsystem<NodeConfigurationSubsystem>();
+        if (!node_configuration_subsystem)
+        {
+          return make_error(
+            HTTP_STATUS_INTERNAL_SERVER_ERROR,
+            ccf::errors::InternalError,
+            "NodeConfigurationSubsystem is not available");
+        }
+
+        auto r = nlohmann::json();
+        r["stop_notice"] =
```

Comment:
I think this should be called "termination_notice" or "terminated" to make it clear it refers to SIGTERM.

Reply:
We call this stop internally though, and we could choose to interpret other signals this way.